### ISSUE TITLE
Issue Using WebWalletLoginButton

### ISSUE THREAD

**Issue by PaulWCZ** [ElrondNetwork/dapp-template#46]
I got many problems with the WebWalletLoginButton:
1-When you are already connected, and logged out then you log in another time, you don't have to scan the maiar QR code or to give the PEM, so you are not really connected. It is like the WebWalletLogin keeps in memory the previous account that log in. The page on the picture, appeared and automatically you are connected
![IMG_F05B9424E1DC-1](https://user-images.githubusercontent.com/96685330/155107922-890aba10-7ee9-418f-9238-53f6091a4a73.jpeg)

2-If I connect through maiar on the web wallet, when I sign a transaction I got an error (see the pictures)
![IMG_9558](https://user-images.githubusercontent.com/96685330/155107771-e56eec25-1eb3-4e99-9b1f-8484609b6d1e.PNG)

3-Another issue but this time with the WalletConnectLoginButton, When you are on phone and wanted to connect through the maiar app directly, you can't click on the button on the pop up, moreover the page freeze.
![IMG_9564](https://user-images.githubusercontent.com/96685330/155107810-42b9dc75-86e9-4449-bf61-c0629057d383.PNG)

Step to reproduce:
1-Connect through the webwallet login button, Disconnect and then reconnect

2-Connect through the web wallet login button, select maiar, then do a Ping transaction

3-Go on your phone on the local server, connect through Maiar button

Sorry for my English, it's definitely not my mother tongue

**Comment by janniksam** [comment]
> Can you try with the latest version? is this still an issue?

I don't have the balance to test it right now, but according to @juliancwirko number 2 is still an issue in this template. We're having the same problems over @ https://github.com/ElrondDevGuild/nextjs-dapp-template/issues/16

**Comment by juliancwirko** [comment]
Yes, to add to that, I tested https://dapp-template.elrond.com The flow: 

1. Choose Web Wallet (login)
2. On the Web Wallet side, use Maiar app
3. Redirect to the dapp-demo app
4. Ping
5. Sign in using Maiar App when on the Web Wallet page
6. Web wallet displays `Rejected by user`

**Comment by janniksam** [comment]
Same problem on devnet.maiarexchange.com.

![image](https://user-images.githubusercontent.com/15337798/200563256-755eb2d0-fdc0-493b-bdb6-09a17ce1e00b.png)

Bulk signing works fine, single transactions don't.

**Comment by MauricioHernanCabrera** [comment]
any solution?

**Comment by icegriffinguru** [comment]
same here :(

**Comment by CiprianDraghici** [comment]
This is a known issue. It will be fixed in the new web wallet version

**Comment by CiprianDraghici** [comment]
I will close the issue as the new web-wallet will be released soon
